src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -7,8 +8,18 @@
 from IntakePipeline.fhir import read_fhir_bundles, upload_bundle_to_fhir_server
 
 
+def get_required_config(varname: str) -> str:
+    """Grab a required config val and throw an exception if it's not present"""
+    if varname not in os.environ:
+        raise Exception(f"Config value {varname} not found in the environment")
+    return os.environ[varname]
```

Comment:
Wound up copying this for my `add_patient_identifier` code. Works like a charm. I moved it into a `utils.py` file since I wanted to access it in some different ways, but otherwise absolutely no issues with it.